src/main/java/ui/components/pickers/PickerMilestone.java

```diff
@@ -23,8 +23,8 @@
     private static final int BIG_LABEL_FONT = 16;
 
     private boolean isSelected = false;
-    private boolean isMatching = false;
     private boolean isExisting = false;
+    private boolean isMatching = true;
```

Comment:
Technically it shouldn't make a difference, as we can always set it later. But `MilestonePickerState` now assumes that they all start with `isMatching`, and does its filtering from there.

I guess `MilestonePickerState` should manually set `isMatching` to true before filtering? Then this initialisation can be false to be consistent with others.